pyanaconda/dbus/typing.py

```diff
@@ -149,24 +149,39 @@ def _get_basic_type(type_hint):
     @staticmethod
     def _is_container_type(type_hint):
         """Is it a container type?"""
-        # Try to get the "base" type of the container type.
+        return DBusType._get_container_base_type(type_hint) is not None
```

Comment:
why not just `bool(DBusType._get_container_base_type(type_hint))` ? that's  what i had in my version before i made it better ;)

Reply:
Version with `bool` is also acceptable, it is just a question of personal preference in this case :-)